custom_reports/object_management_api/definitions/product_custom_dimension.yaml

```
@@ -0,0 +1,31 @@
+type: object
+required:
+  - type
+  - attributes
+  - id
+properties:
+  type:
+    type: string
+    description: Type of the object.
+    enum:
+      - ProductCustomDimension
+  id:
+    type: string
+    description: ID of a Product Custom Dimension.
+    readOnly: true
+    format: uuid
+    example: 677fe0d3-b4f7-43ed-852d-1a9e7b8e59fb
+  attributes:
+    required:
+      - name
+      - website_id
+    properties:
+      website_id:
+        type: string
+        format: uuid
+        description: ID of the website.
+        example: dde2dc68-b6bb-45ee-9ac2-868b6181e33f
+      name:
+        type: string
+        description: Name of the product custom dimension.
+        example: Visitor hair color
```

```suggestion
        example: Color
```
? I don't think shops often have products with `Visitor hair color` property

---

only for readheads 😆 :trollface:

---

I changed this already except i do not push changes ... :/